### PR TITLE
Replace EMAIL_REGEX with email-validator package

### PR DESCRIPTION
**Pull request by siraben**
Rebased on latest master.

### CODE REVIEW

File: src/client/routes/manage/CreateSponsor.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
 import Spinner from '../../components/Loading/Spinner';
 import STRINGS from '../../assets/strings.json';
 
-const EMAIL_REGEX = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+import * as EmailValidator from 'email-validator';
```
Comment (leonm1) [inline src/client/routes/manage/CreateSponsor.tsx:18]: This should go on line 3. I believe the import sorter will put it in the right place if you eliminate the whitespace on line 17.

**Review by leonm1 — CHANGES_REQUESTED** [review]

**Review by leonm1 — APPROVED** [review]

**Comment by leonm1** [comment]
Thanks for the PR!

**Comment by siraben** [comment]
@bencooper222 was it redundant?

**Comment by siraben** [comment]
Rebased on master

**Review by bencooper222 — APPROVED** [review]
lgtm

**Comment by aadibajpai** [comment]
@siraben it should be used here as well https://github.com/VandyHacks/vaken/blob/main/scripts/makeOrganizer.ts#L5 I guess

**Comment by siraben** [comment]
Thanks, will look into this.

**Comment by aadibajpai** [comment]
> Thanks, will look into this.

in fact, https://github.com/VandyHacks/vaken/search?q=EMAIL_REGEX

**Comment by siraben** [comment]
😱 DESTROY ALL THE EMAIL REGEX

**Pull request merged**